Formula/yt-dlp.rb

```diff
@@ -1,17 +1,14 @@
 class YtDlp < Formula
+  include Language::Python::Virtualenv
+
   desc "Fork of youtube-dl with additional features and fixes"
   homepage "https://github.com/yt-dlp/yt-dlp"
-  url "https://github.com/yt-dlp/yt-dlp/archive/2021.08.10.tar.gz"
-  sha256 "d847e32bc18767be2f27183f5da2ba05d0142e194e95ac6b3e69494353ee366b"
+  url "https://files.pythonhosted.org/packages/ee/59/d763a51fa975639946c327ffbf85e109dbaa87f8aa5cae54a4ee5d09593c/yt-dlp-2021.8.10.tar.gz"
+  sha256 "8da1bf4dc4641d37d137443c4783109ee8393caad5e0d270d9d1d534e8f25240"
   license "Unlicense"
+  revision 1
```

Comment:
Shouldn't the install be the same for users?

Reply:
Makes sense, will remove the revision